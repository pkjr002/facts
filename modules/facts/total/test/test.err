  File "/scratch/rk509/test.1667525579/total_workflow.py", line 61
    def TotalSamplesInWorkflows(directory, wfcfg_file="workflows.yml", pyear_start, pyear_end, pyear_step, chunksize, workflow=[""], scale=[""]):
                                                                                    ^
SyntaxError: non-default argument follows default argument
