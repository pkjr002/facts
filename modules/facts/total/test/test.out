Initiating test in /scratch/rk509/test.1667525579...

Skipping data extraction...

Executing workflow...
python total_workflow.py --pyear_start 2020 --pyear_end 2100 --pyear_step 10 --directory test/data --workflows --workflow wf2

Collecting output files...

Cleaning up...
